src/main/drivers/serial_uart_stm32f30x.c

```diff
@@ -38,6 +40,31 @@
 #include "serial_uart.h"
 #include "serial_uart_impl.h"
 
+typedef struct uartPinPair_s {
+    ioTag_t rx;
+    ioTag_t tx;
+} uartPinPair_t;
+
+#define UART_RX_BUFFER_SIZE 256
+#define UART_TX_BUFFER_SIZE 256
+
+typedef struct uartDevice_s {
+    USART_TypeDef* dev;
+    uartPort_t port;
```

Comment:
Year, ones outside the uartDevice_s will be deleted.

What I don't like is that [rt]xBuffer for non-configured UARTs are wasted...